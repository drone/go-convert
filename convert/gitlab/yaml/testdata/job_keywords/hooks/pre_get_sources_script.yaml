# https://docs.gitlab.com/ee/ci/yaml/#hookspre_get_sources_script

job1:
  hooks:
    pre_get_sources_script:
      - echo 'hello job1 pre_get_sources_script'
  script: echo 'hello job1 script'