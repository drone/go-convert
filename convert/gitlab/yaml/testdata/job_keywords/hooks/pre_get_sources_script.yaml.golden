job1:
    hooks:
        pre_get_sources_script:
            - echo 'hello job1 pre_get_sources_script'
    script:
        - echo 'hello job1 script'
